charts/backbeat/values.yaml

```diff
@@ -6,7 +6,7 @@ orbit:
 
 image:
   repository: zenko/backbeat
-  tag: 0.2.4
+  tag: 8.0.0
```

Comment:
Was this intended to bump to 8.0.0?

Reply:
Yeah we are starting from 8 for any new images =)